notes_app/notes/tests.py

```diff
@@ -155,3 +155,291 @@ def test_delete_detail(self):
         self.assertHttpMethodNotAllowed(self.api_client.delete(
             self.api_detail, format='json',
             authentication=self.get_credentials()))
+
+
+# User Auth Tests ================================
+class UserResourceTest(ResourceTestCaseMixin, TestCase):
```

Comment:
Could you please create a separate file for user tests? `test_user_resource.py` for example